lib/minitest/reporters/spec_reporter.rb

```diff
@@ -28,10 +38,20 @@ def report
 
       def record(test)
         super
-        test_name = test.name.gsub(/^test_: /, 'test:')
-        print pad_test(test_name)
-        print_colored_status(test)
-        print(" (%.2fs)" % test.time) unless test.time.nil?
+        if options[:show_order] == :before
+           print_colored_status(test) if options[:show_status]
```

Comment:
It's indented due to if/else statement.  I'm separating and retaining the current default output in the else statement.  I did not wish to change that part.